src/main/java/frc/robot/commands/LEDCommand.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ESP;
+import frc.robot.subsystems.Shooter;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class LEDCommand extends Command {
+  /** Creates a new DigitalCommand. */
+  ESP esp; 
+  Shooter shooter; 
+  
+  public LEDCommand(Shooter shooter, ESP esp) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    this.esp = esp; 
+    this.shooter = shooter;   
+    addRequirements(esp);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    esp.setLED(true); 
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    SmartDashboard.putBoolean("topSensor", shooter.topPieceSensorActive()); 
```

Comment:
this shouldn't be where smart dashboard puts data, that should be from the subsystem and I believe it already did that

Reply:
Removed